draft-ietf-jsonpath-base.md

```diff
@@ -246,7 +246,11 @@ Unicode Scalar Value:
   E000 to 10FFFF. JSON values of type string are sequences of Unicode scalar values.
 
 Singular Path:
-: A JSONPath expression built from selectors which each select at most one node.
+: A JSONPath expression built from appenders which each select at most one node.
```

Comment:
This sentence demonstrates a weakness in the "appender"/"selector" terminology. In plain English, it is more natural to talk about selectors selecting and appenders appending.

Reply:
appender is placeholder terminology.

But to the point: Just because an oven bakes, we don't have to call it a baker.

Reply:
> appender is placeholder terminology.

Hmmm. It looks concrete to me. Perhaps we should bring back some of the placeholders until we can think of better terms.

> 
> But to the point: Just because an oven bakes, we don't have to call it a baker.

Sure, but if we choose a term like baker, it's hard to avoid all baking connotations. In the current draft, we have selectors which select and that terminology is easy to understand and use. We need something similarly readable for these new terms.